src/protocol/cdp.ts

```diff
@@ -20,6 +20,8 @@ import type Protocol from 'devtools-protocol';
 
 import type {BrowsingContext, JsUint} from './webdriver-bidi.js';
 
+export type EventNames = 'cdp' | Event['method'];
```

Comment:
It seem we never fix this type this should not include `cdp` as `cdp` is threaded as a module and should be added to the module enum.